homeassistant/components/calendar/__init__.py

```diff
@@ -70,10 +70,13 @@ def normalize_event(event):
 
```

Comment:
Would like to add created and updated attributes. It's in Z format: 2011-08-17T23:20:57.000Z